### ISSUE TITLE
SequenceEncoder is not working in JellyBean

### ISSUE THREAD

**Issue by Shvet** [jcodec/jcodec#131]
I have used SequenceEncoder to make mp4 from images but it is showing only 1 image for full video even if  have more then 1 image.

I am using 0.1.9 version of jcodec.Is there any solution for this? I have checked it is running encodeNativeFrame method for all images i have added.

**Comment by devinmanzo** [comment]
@Shvet i'm getting the same error did you figure it out?

**Comment by Shvet** [comment]
@devinmanzo Nope, couldnt get solution and there is no response from @jcodec too. I just removed library from project and used POI for android. I also try to made library from POI java by copying their classes but it failed too, Check [https://github.com/Shvet/POIAndroid](POIAndroid), there are lots of errors and classes left to add in it. If you want to use it and modify it, feel free to clone.

**Comment by devinmanzo** [comment]
@Shvet That's ok thanks anyway. A previous user from years ago had this issue as well. But he moved on to converting his images to gifs. I did so as well, it serves my needs for now. i will use this method until i'm able to fix or understand this issue. Either that or i'll learn to use ffmpeg with android.
